android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -63,7 +63,7 @@
     // Query KMW if a given keyboard uses chiral modifiers.
     function setIsChiral(keyboardProperties) {
       var name = typeof(keyboardProperties.internalName) == "undefined" ? keyboardProperties.keyboardName : keyboardProperties.internalName;
-      var isChiral = tavultesoft.keymanweb.isChiral(name);
+      var isChiral = keymanweb.isChiral(name);
```

Comment:
I assume `KeymanWeb` is a global here?

Reply:
`keymanweb`- autocorrect messed it up.

Reply:
good catch. I need to change those to `kmw`.